h2/settings.py

```diff
@@ -253,7 +253,18 @@ def __iter__(self):
     def __len__(self):
         return len(self._settings)
 
-
+    def __eq__(self, other):
+        if isinstance(other, Settings):
+            return self._settings == other._settings
+        else:
+            return False
+            
+    def __ne__(self, other):
+        if isinstance(other, Settings):
```

Comment:
I’d push for

```python
return not self == other
```

unless there’s a good reason not to, as otherwise we’re duplicating logic from the `__eq__` method.